src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -144,6 +145,40 @@ public static class Metrics
         public const string AdditionalSources = "OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES";
     }
 
+    /// <summary>
+    /// Configuration keys for logs exporter
```

Comment:
```suggestion
    /// Configuration keys for logs.
```

Reply:
We have the same pattern for traces and logs, will get that modified too.